### ISSUE TITLE
Bug: Title text slightly moves up without actionable contents in narrow width

### ISSUE THREAD

**Issue by 0x5bfa** [CommunityToolkit/Windows#219]
### Describe the bug

As `@leoshusar` said before, title text moves up without contents, or contents moves down without title text. Refer to the screenshots section

You should add Visibility=Collapsed to both of Contents content presenter and Title content presenter change the visibility state when the value is set (OnContentChanged and OnHeaderChanged).

https://github.com/CommunityToolkit/Windows/blob/a79f452ddd3a211a15ea7336fb8bb40a031ae107/components/SettingsControls/src/SettingsCard/SettingsCard.xaml#L392-L433

https://github.com/CommunityToolkit/Windows/blob/a79f452ddd3a211a15ea7336fb8bb40a031ae107/components/SettingsControls/src/SettingsCard/SettingsCard.xaml#L435-L440

This margin is the cause of this bug.
https://github.com/CommunityToolkit/Windows/blob/a79f452ddd3a211a15ea7336fb8bb40a031ae107/components/SettingsControls/src/SettingsCard/SettingsCard.xaml#L109

### Steps to reproduce

```text
1. Add SettingsCard without contents.
2. Make control narrower.
3. See what's happening.
```

### Expected behavior

Do not move up or down.
Be fixed on the same place.

### Screenshots

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/4234d203-c7c1-4df8-add0-c4ed865c7c7c)

![image](https://github.com/CommunityToolkit/Labs-Windows/assets/62196528/1f15fa04-c94e-452d-98f0-8ed6c11d4f0d)

### Code Platform

- [X] UWP
- [X] WinAppSDK / WinUI 3
- [ ] Web Assembly (WASM)
- [ ] Android
- [ ] iOS
- [ ] MacOS
- [ ] Linux / GTK

### Windows Build Number

- [ ] Windows 10 1809 (Build 17763)
- [ ] Windows 10 1903 (Build 18362)
- [ ] Windows 10 1909 (Build 18363)
- [ ] Windows 10 2004 (Build 19041)
- [ ] Windows 10 20H2 (Build 19042)
- [ ] Windows 10 21H1 (Build 19043)
- [X] Windows 11 21H2 (Build 22000)
- [ ] Other (specify)

### App minimum and target SDK version

- [ ] Windows 10, version 1809 (Build 17763)
- [ ] Windows 10, version 1903 (Build 18362)
- [X] Windows 10, version 1909 (Build 18363)
- [ ] Windows 10, version 2004 (Build 19041)
- [ ] Other (specify)

### Visual Studio Version

Preview

### Help us help you

No, I'm unable to contribute a solution.

**Comment by niels9001** [comment]
@0x5bfa Maybe setting the RowSpacing instead of a `Margin` is the easier way to fix this?

**Comment by niels9001** [comment]
Fixed in #252
